app/src/main/assets/templates/cooking-recipe.md

```diff
@@ -0,0 +1,20 @@
+# Title
```

Comment:
All of these template files have been moved to resources instead of strings in the code